contracts/Bridge.sol

```diff
@@ -106,8 +108,19 @@ contract Bridge is Pausable, AccessControl, SafeMath {
         return (_relayerBit(relayer) & uint(proposal._yesVotes)) > 0;
     }
 
+    function _msgSender() internal override view returns (address payable signer) {
+        signer = msg.sender;
+        if (isValidForwarder[signer]) {
+            bytes memory data = msg.data;
+            uint256 length = msg.data.length;
+            assembly {
```

Comment:
if assembly is used anyway, then we should use the same implementation as here:
https://github.com/opengsn/gsn/blob/master/packages/contracts/src/BaseRelayRecipient.sol#L36
because it is cheaper atleast, by not copying everything to memory and instead extracting the sender right from the calldata.

Reply:
make sense :)